src/Resources/doc/index.rst

```diff
@@ -65,18 +65,74 @@ file by their respective environment variables:
 Configuration
 -------------
 
-This bundle doesn't require any configuration. But, you *can* configure
-the root namespace that is used to "guess" what classes you want to generate:
+This bundle doesn't require any configuration. But, you *can* override the default
+configuration:
 
 .. code-block:: yaml
 
-    # config/packages/dev/maker.yaml
-    # create this file if you need to configure anything
-    maker:
-        # tell MakerBundle that all of your classes live in an
-        # Acme namespace, instead of the default App
-        # (e.g. Acme\Entity\Article, Acme\Command\MyCommand, etc)
-        root_namespace: 'Acme'
+    # config/packages/maker.yaml
+    when@dev:
+        maker:
+            root_namespace: 'App'
+            generate_final_classes: true
+            generate_final_entities: false
+
+root_namespace
+~~~~~~~~~~~~~~
+
+**type**: ``string`` **default**: ``App``
+
+The root namespace used when generating all of your classes
+(e.g. ``App\Entity\Article``, ``App\Command\MyCommand``, etc). Changing
+this to ``Acme`` would cause MakerBundle to create new classes like
+(e.g. ``Acme\Entity\Article``, ``Acme\Command\MyCommand``, etc).
+
+generate_final_classes
+~~~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``true``
+
+By default, MakerBundle will generate all of your classes with the
+``final`` PHP keyword except for doctrine entities. Set this to ``false``
+to override this behavior for all maker commands.
+
+See https://www.php.net/manual/en/language.oop5.final.php
+
+.. code-block:: php
+
+    final class MyVoter
+    {
+        ...
+    }
+
+.. versionadded:: 1.60.0
+
+    ``generate_final_classes`` was introduced in MakerBundle v1.60
+
+
+generate_final_entities
+~~~~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+By default, MakerBundle will not generate any of your doctrine entity
+classes with the ``final`` PHP keyword. Set this to ``true``
+to override this behavior for all maker commands that create
+entities.
+
+See https://www.php.net/manual/en/language.oop5.final.php
+
+.. code-block:: php
+
+    #[ORM\Entity(repositoryClass: TaskRepository::class)]
+    class Task extends AbstractEntity
+    {
+        ...
+    }
+
+.. versionadded:: 1.60.0
+
+    ``generate_final_entities`` was introduced in MakerBundle v1.60
```

Comment:
```suggestion
    ``generate_final_entities`` was introduced in MakerBundle 1.60.
```

should always end with a dot and I am not sure the `v` is needed

Reply:
note to self: when updating this, `1.60` needs to be bumped to `61` or higher - we havent merged the feature yet.